frontend/app/types/card.ts

```diff
@@ -1,8 +1,10 @@
 export interface CardData {
-  id: number;
-  title: string;
-  content: string;
+  id: string;
+  meaning: string;
+  explanation: string;
+  examples: string;
   isFavorite?: boolean;
+  text: string;
```

Comment:
let's move the text to the second spot after the ID. It's the second most important field